### PR TITLE
Legacy/pinsec/hardware: updated (REVIEW/REWORD requested)

### PR DESCRIPTION
**Pull request by dlmiles**
Browsable preview at: https://dlmiles.github.io/SpinalDoc-RTD/docs_multi/html/vTEST/index.html

https://dlmiles.github.io/SpinalDoc-RTD/docs_multi/html/vTEST/SpinalHDL/Legacy/pinsec/hardware.html

See review marker looking for assistance to reword this area.  Thanks

### CODE REVIEW

**Review by dlmiles — COMMENTED** [review]
Please rephrase this to make clearer the meaning.  Thanks

File: source/SpinalHDL/Legacy/pinsec/hardware.rst [hunk 1]
```diff
@@ -28,23 +28,23 @@ The RISCV is a 5 stage pipelined CPU with following features :
 AXI4
 ----
 
-As previously said, Pinsec integrate an AXI4 bus fabric. AXI4 is not the easiest bus on the Earth but has many advantages like :
+As previously said, Pinsec integrate an AXI4 bus fabric. AXI4 is not the easiest bus to work with but has many advantages like :
```
Comment (andreasWallner) [inline source/SpinalHDL/Legacy/pinsec/hardware.rst:31]:
```suggestion
As previously said, Pinsec integrates an AXI4 bus fabric. AXI4 is not the easiest bus to work with but has many advantages like:
```

File: source/SpinalHDL/Legacy/pinsec/hardware.rst [hunk 2]
```diff
@@ -28,23 +28,23 @@ The RISCV is a 5 stage pipelined CPU with following features :
...
 * Introducing an Axi4-Shared variation where a new ARW channel is introduced to replace AR and AW channels. This solution divide resources usage by two for the address decoding and the address arbitration.
-* Depending the interconnect implementation, if masters doesn't use the R/B channels ready, this path will be removed until each slaves at synthesis, which relax timings.
-* As the AXI4 spec suggest, the interconnect can expand the transactions ID by aggregating the corresponding input port id. This allow the interconnect to have an infinite number of pending request and also to support out of order completion with a negligible area cost (transaction id expand).
+* Depending the interconnect implementation, if masters doesn't use the R/B channels ready, this path will be removed until each slaves at synthesis, which relax timings.  REVIEWME
+* As the AXI4 spec suggest, the interconnect can expand the transactions ID by aggregating the corresponding input port ID. This allow the interconnect to have an infinite number of pending request and also to support out of order completion with a negligible area cost (transaction ID expand).
```
Comment (andreasWallner) [inline source/SpinalHDL/Legacy/pinsec/hardware.rst:47]:
```suggestion
* As the AXI4 spec suggest, the interconnect can expand the transactions ID by aggregating the corresponding input port ID. This allows the interconnect to have an infinite number of pending request and also to support out of order completion with a negligible area cost (transaction ID expand).
```

File: source/SpinalHDL/Legacy/pinsec/hardware.rst [hunk 3]
```diff
@@ -28,23 +28,23 @@ The RISCV is a 5 stage pipelined CPU with following features :
...
+From an Area utilization perspective, AXI4 is for sure not the lightest solution, but some techniques could dramatically reduce that concern :
 
 
-* Using Read-Only/Write-Only AXI4 variations where it's possible
+* Using Read-Only/Write-Only AXI4 variations where that is possible
 * Introducing an Axi4-Shared variation where a new ARW channel is introduced to replace AR and AW channels. This solution divide resources usage by two for the address decoding and the address arbitration.
-* Depending the interconnect implementation, if masters doesn't use the R/B channels ready, this path will be removed until each slaves at synthesis, which relax timings.
-* As the AXI4 spec suggest, the interconnect can expand the transactions ID by aggregating the corresponding input port id. This allow the interconnect to have an infinite number of pending request and also to support out of order completion with a negligible area cost (transaction id expand).
+* Depending the interconnect implementation, if masters doesn't use the R/B channels ready, this path will be removed until each slaves at synthesis, which relax timings.  REVIEWME
```
Comment (andreasWallner) [inline source/SpinalHDL/Legacy/pinsec/hardware.rst:46]:
```suggestion
* Timing relaxation is possible depending the interconnect implementation, and if masters never stall the R/B channel (RREADY and BREADY are strapped to 1). Both xREADY signals can be removed by synthesis in this case, relaxing timings.
```

**Review by andreasWallner — CHANGES_REQUESTED** [review]
LGTM, just fixed two more spelling mistakes and proposed an updated wording to your REVIEWME

**Comment by dlmiles** [comment]
SQUISH with log message cleanup and rebase ready for merge.

Tidy commit and messages :)

**Comment by andreasWallner** [comment]
Thanks!
